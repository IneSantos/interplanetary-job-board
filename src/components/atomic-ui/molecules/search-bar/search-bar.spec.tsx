import React from "react";
import { render } from '@testing-library/react';
import SearchBar from "./search-bar";

describe("SearchBar atom tests", () => {
  test(`SearchBar component render`, () => {
    const { container } = render(
      <SearchBar searchInput="This is a input" handleSearch={jest.fn()}/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
