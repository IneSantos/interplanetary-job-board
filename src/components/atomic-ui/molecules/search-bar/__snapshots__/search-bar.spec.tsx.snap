// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar atom tests SearchBar component render 1`] = `
<div
  class="search"
>
  <button
    class="search-button"
    type="submit"
  >
    <i
      class="fas fa-search"
    />
  </button>
  <input
    class="search-term"
    placeholder="What are you looking for?"
    type="text"
    value="This is a input"
  />
</div>
`;
